pwndbg/commands/peda.py

```diff
@@ -19,15 +19,21 @@ def getfile() -> None:
     print(repr(pwndbg.auxv.get().AT_EXECFN))
 
 
-parser = argparse.ArgumentParser(description="Continue execution until an address or function.")
-parser.add_argument("target", type=str, help="Address or function to stop execution at")
+parser = argparse.ArgumentParser(
+    description="Continue execution until an address or function."
+)
+parser.add_argument("target", type=int, help="Address or function to stop execution at")
 
 
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.NEXT)
 def xuntil(target) -> None:
+    running = pwndbg.gdblib.proc.alive
+    if not running:
+        print(message.notice("Process is not running. Starting process..."))
+        gdb.execute("starti", from_tty=False, to_string=True)
+        target = pwndbg.gdblib.functions.rebase(target)
```

Comment:
We don't want to use `gdblib` in `commands/*` since this will work only in GDB and not in LLDB

Reply:
This will not work if the process died and we restart it with `xuntil main` - at this point the symbol will already be rebased and we will rebase it again